charts/rd-professional-api/values.yaml

```diff
@@ -18,10 +18,14 @@ java:
   keyVaults:
     rd:
       secrets:
-        - professional-api-POSTGRES-PASS
-        - ApplicationInsightsInstrumentationKey
-        - LD-SDK-KEY
-        - professional-api-s2s-secret
+        - name: professional-api-POSTGRES-PASS
+          alias: spring.datasource.password
+        - name: ApplicationInsightsInstrumentationKey
+          alias: azure.application-insights.instrumentation-key
```

Comment:
do we need alias mandatory?

Reply:
Yes, this will replace the use of `bootstrap.yaml`